app/grandchallenge/notifications/models.py

```diff
@@ -158,6 +158,11 @@ def send(type, **kwargs):  # noqa: C901
                 ]
             else:
                 receivers = followers(target)
+        elif (
+            type == NotificationType.NotificationTypeChoices.ACCESS_REQUEST
+            and target._meta.model_name == "algorithm"
+        ):
+            receivers = followers(target, flag="access_request")
```

Comment:
This is unfortunately necessary because `followers(target, flag='')` unexpectedly returns not only follows with a flag of '' but also follows with a filled flag (i.e. 'job-active' or 'job-inactive')....